kosaraju_Algorithm.cpp

```diff
@@ -0,0 +1,103 @@
+//Strongly connected components(KOSARAJU ALGORITHM)
+#include <iostream>
+#include <vector>
+#include <stack>
+#include <unordered_set>
+using namespace std;
+
+void dfs(vector<int>* edges, int start, unordered_set<int> &visited, stack<int> &finishStack) { //dfs to store vertices in a stack as per their finish time
+	visited.insert(start);
+	for (int i = 0; i < edges[start].size(); i++) {
+		int adjacent = edges[start][i];
+		if (visited.count(adjacent) == 0) {
+			dfs(edges, adjacent, visited, finishStack);
+		}
+	}
+	finishStack.push(start);
```

Comment:
correct the indentation.keep four spaces instead of tab

Reply:
@asha15 done  please review it